modules/oe_content_timeline_field/src/Plugin/Field/FieldWidget/TimelineFieldWidget.php

```diff
@@ -67,4 +71,55 @@ public function massageFormValues(array $values, array $form, FormStateInterface
     return $values;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function errorElement(array $element, ConstraintViolationInterface $violation, array $form, FormStateInterface $form_state) {
+    if (!empty($violation->arrayPropertyPath) && $sub_element = NestedArray::getValue($element, $violation->arrayPropertyPath)) {
+      return $sub_element;
+    }
+    return $element;
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * Override the parameters to use the form element labels.
+   */
+  public function flagErrors(FieldItemListInterface $items, ConstraintViolationListInterface $violations, array $form, FormStateInterface $form_state) {
+    /** @var \Symfony\Component\Validator\ConstraintViolationInterface $violation */
+    foreach ($violations as $offset => $violation) {
+      $initial_parameters = $violation->getParameters();
+      $parameters = $initial_parameters;
```

Comment:
```suggestion
       $parameters = $initial_parameters = $violation->getParameters();
```
could be like that. But it is optional comment.